scripts/generate-rpc-configs.js

```diff
@@ -6,7 +6,47 @@ const GITHUB_API_URL = 'https://api.github.com/repos/wevm/viem/contents/src/chai
 const GITHUB_RAW_BASE_URL =
   'https://raw.githubusercontent.com/wevm/viem/main/src/chains/definitions';
 const OUTPUT_FILE = path.join(process.cwd(), '../public/configs/rpc.json');
-const GITHUB_TOKEN = ''; // Add your GitHub token here if necessary
+
+// see https://github.com/dydxprotocol/v4-web/blob/main/src/lib/wagmi.ts#L111-L136
+const ChainId = {
+  ETH_MAINNET: '1',
+  ETH_SEPOLIA: '11155111',
+  POLYGON_MAINNET: '137',
+  POLYGON_MUMBAI: '80002',
+  OPT_MAINNET: '10',
+  OPT_SEPOLIA: '11155420',
+  ARB_MAINNET: '42161',
+  ARB_SEPOLIA: '421614',
+  BASE_MAINNET: '8453',
+  BASE_SEPOLIA: '84532',
+};
+
+const getAlchemyRPCUrls = (chainId) => {
+  switch (chainId) {
+    case ChainId.ETH_MAINNET:
+      return `https://eth-mainnet.g.alchemy.com/v2`;
```

Comment:
do we need an API key here? like this: https://github.com/dydxprotocol/v4-web/blob/634a4b1752ddc127aa9f13e0fb99b489e1f6208c/src/lib/wagmi.ts#L111-L136

Reply:
@yogurtandjam nope, apiKey is private so will have to be injected by mobile clients at build-time. The abacus and mobile work will cover this